src/shared/pretty-print.c

```diff
@@ -170,7 +188,7 @@ int cat_files(const char *file, char **dropins, CatFlags flags) {
         int r;
 
         if (file) {
-                r = cat_file(file, false);
+                r = cat_file(file, false, flags);
```

Comment:
if you touch this, please update to current coding style, i.e. replace `false` by `/* newline= */ false` 

(or better, merge this into the flags param

Reply:
As discussed above, I don't think merging this with flags is advised. I added the comment.